app/controllers/claims/schools/claims_controller.rb

```diff
@@ -21,7 +21,18 @@ def create
 
   def show; end
 
-  def check; end
+  def check
+    last_mentor_training = @claim.mentor_trainings.order_by_mentor_full_name.last
+
+    @back_path = edit_claims_school_claim_mentor_training_path(
+      @school,
+      @claim,
+      last_mentor_training,
+      params: {
+        claim_mentor_training_form: { hours_completed: last_mentor_training.hours_completed },
+      },
+    )
+  end
```

Comment:
This solves a bug where the back link on the check page would go to the claim mentors list. Now it goes to the last mentor_training#hours_completed record